src/main/protractor/po/profileSettingsPage.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
this is profile settings page?
this page has the contact information, check please